packages/datadog-instrumentations/src/body-parser.js

```diff
@@ -16,11 +22,11 @@ function publishRequestBodyAndNext (request, next) {
 addHook({
   name: 'body-parser',
   file: 'lib/read.js',
-  versions: ['>=1']
+  versions: ['>=1.4.0']
```

Comment:
we can't drop support for body-parse<1.4.0 that easily

Reply:
Indeed. What is the reasoning behind this change?

Reply:
We realized it was not even working for versions under 1.4.0 before, and since it wasn't tested like we discussed below, we didn't realize it until now